packages/core/src/checkout/checkout-service.ts

```diff
@@ -1399,6 +1404,37 @@ export default class CheckoutService {
         return this._dispatch(action, { queueId: 'extensions' });
     }
 
+    /**
+     * Registers command handlers. Works on both host and extension sides.
+     *
+     * @alpha
+     * @param handlers - a set of handlers for checkout extension commands
+     * @returns A promise that resolves to the current state.
+     */
+    handleExtensionCommand(handlers: ExtensionCommandHandlers): Promise<CheckoutSelectors> {
+        const action = this._extensionActionCreator.handleExtensionCommand(handlers);
```

Comment:
I have changed it. Because I couldn't find another way to setup extensions inside the messenger, I'm requesting it as a parameter here.
```
handleExtensionCommand(extensions: Extension[], handlers: ExtensionCommandHandlers): void
```

To remove handlers, simply use this function again with a new set of desired handlers, which I recommend as an efficient way to set up multiple handlers at once. Do we want to add/remove hander one at a time?

Reply:
> To remove handlers, simply use this function again with a new set of desired handlers

I can't see `ExtensionMessenger` making any `window.removeEventListener` call? 🤔 We need a way to remove command handlers to avoid potential memory leaks. For example, we could end up with a leak if a component is unmounted and it still has a function registered as a command handler.